README.md

```diff
@@ -142,8 +142,8 @@ In order to publish the webhook endpoints and the resulting vocabs we recommend
     RewriteRule (.*)/images/(.*)           http://10.0.0.1/$1/images/$2 [P,L]
     
     RewriteCond %{REQUEST_METHOD} "=POST"
```

Comment:
Is this really doing what it is supposed to do? Because the original issue arose with a `GET` request

Reply:
The quotes in `"=POST"` are missing in the actual apache config, that'S why your `GET` request got also redirected.